parse/src/main/java/org/dst/parse/Operate.g4

```diff
@@ -0,0 +1,13 @@
+grammar Operate;
```

Comment:
It's better to name this file `DstNewSQLGrammar`.

Reply:
this is not only contain sqlgrammer,but has operate grammar,DstNew SQLGrammar is right?